### DISCUSSION TITLE
A black outline is not black when the grid size to 1

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#488] [Q&A]
![image](https://user-images.githubusercontent.com/36077492/225360467-74aedf22-1784-49e6-b6c6-db3a09cb1b2d.png)

Here I am trying pixel art style. I set the outline width to 1 pixel and color to black. But the render result is not pure black. How to let the outline be exactly the color I set independent from the grid size?

**Comment by HSDSZ** [comment]
Strange!

If I set the outline width scale to 2 pixel. The outline become black even though the rendered outline is still 1 pixel width
![image](https://user-images.githubusercontent.com/36077492/225361725-6df1deff-2e16-4b28-bc37-c112c5f509e6.png)

**Comment by pragma37** [comment]
Sorry for the late reply.
This is a known issue, I could fix it but I'm not sure if it's a good idea since it would change how existing materials look like. 🤔

**Reply by HSDSZ** [reply]
Could I know how to fix this? I could fix this in my own version

**Reply by pragma37** [reply]
I've recently updated the `line_expand` function, so I included this fix as well.
I hope it doesn't bother anyone. 🙄
(It's in the Development branch only for now)
